k8s/namespaces/rd/rd-fee-pay-ref-data-load/aat-00.yaml

```diff
@@ -0,0 +1,9 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: rd-fee-pay-ref-data-load
+spec:
+  values:
+    job:
+      schedule: "0/10 * * * *"
```

Comment:
schedules don't make sense to me ? Are they really run at 10 minute interval in production ? Also if u are running one for 10 minutes and other for 15, u are running them in parallel every half hour. Is this ok ?

Reply:
@adusumillipraveen it is ok for now as this repo was recently created. When the features are built, the team will know the schedule in which they have to run and update them accordingly.

Reply:
If It is not used, then why not run it less frequent ? And its important to make them not clash. If they can run parallel at once, then it can go to common-overlay

Reply:
Updated schedule to run once a day. Job will run 1 hour apart on each cluster.